docs/src/CommutativeAlgebra/ModulesOverMultivariateRings/free_modules.md

```diff
@@ -15,30 +15,30 @@ Pages = ["free_modules.md"]
 
 # Free Modules
 
-In what follows, the expression *free module*  refers to a free module of finite rank
-over a multivariate polynomial ring. More concretely, given a multivariate polynomial ring $R$, 
-the free $R$-modules considered are of type $R^p$, where we think of $R^p$ as a
-free module with a given basis, namely the basis of standard unit vectors.
-Accordingly, elements of free modules are represented by coordinate vectors,
+In this section, the expression *free module*  refers to a free module of finite rank
+over a ring of type `MPolyRing`, `MPolyQuo`, `MPolyLocalizedRing`, or `MPolyQuoLocalizedRing`.
+More concretely, given a ring $R$ of one of these types, the free $R$-modules considered are of
+type $R^p$, where we think of $R^p$ as a free module with a given basis, namely the basis of
+standard unit vectors. Accordingly, elements of free modules are represented by coordinate vectors,
 and homomorphisms between free modules by matrices.
 
 !!! note
     By convention, vectors are row vectors, and matrices operate by multiplication on the right.
 
 ## Types
 
-All OSCAR types for finitely presented modules over multivariate polynomial rings belong to
-the abstract type `ModuleFP{T}`, where `T` is the element type of the polynomial ring.
-For free modules, OSCAR provides the abstract subtype `AbstractFreeMod{T} <: ModuleFP{T}`
-and its concrete descendant `FreeMod{T <: RingElem}`.
+All OSCAR types for the modules considered here belong to the
+abstract type `ModuleFP{T}`, where `T` is the element type of the underlying ring.
+The free modules belong to the abstract subtype `AbstractFreeMod{T} <: ModuleFP{T}`,
+they are modelled as objects of the concrete type `FreeMod{T} <: AbstractFreeMod{T}`.
 
 !!! note
     Canonical maps such us the canonical projection onto a quotient module arise in many 
     constructions in commutative algebra. The `FreeMod` type is designed so that it allows
     for the caching of such maps when executing functions. The `direct_sum` function discussed
     in this section provides an example.
 
-## Constructor
+## Constructors
 
 ```@docs
 free_module(R::MPolyRing, n::Int, name::String = "e"; cached::Bool = false)
```

Comment:
Below in line 53 there is a discussion of `dim(F)` as a synonym to `rank(F)` for a free module `F`. But the dimension of an arbitrary module can also be understood as the dimension of its support, right? Shouldn't we reserve the `dim` command for that?

Reply:
Frankly, I find `dim` here a little strange as well, but it is available in UngradedModules.jl. @jankoboehm ???

Reply:
It is only defined for free modules